<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-IT7JOLC2" failures="2" tests="17" name="com.myproject.tests.ApiListPageUITest" time="26.792" errors="0" timestamp="2025-09-16T03:21:36 IST" skipped="0">
  <testcase classname="com.myproject.tests.ApiListPageUITest" name="tc_api_list_accessible_via_url" time="1.998"/>
  <testcase classname="com.myproject.tests.ApiListPageUITest" name="tc_home_nav" time="3.344"/>
  <testcase classname="com.myproject.tests.ApiListPageUITest" name="tc_products_nav" time="3.046"/>
  <testcase classname="com.myproject.tests.ApiListPageUITest" name="tc_cart_nav" time="1.741"/>
  <testcase classname="com.myproject.tests.ApiListPageUITest" name="tc_signup_nav" time="1.563"/>
  <testcase classname="com.myproject.tests.ApiListPageUITest" name="tc_testcases_nav" time="1.592"/>
  <testcase classname="com.myproject.tests.ApiListPageUITest" name="tc_videos_nav" time="3.503">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.myproject.tests.ApiListPageUITest.tc_videos_nav(ApiListPageUITest.java:67)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- tc_videos_nav -->
  <testcase classname="com.myproject.tests.ApiListPageUITest" name="tc_api_list_title_present" time="0.521"/>
  <testcase classname="com.myproject.tests.ApiListPageUITest" name="tc_feedback_section_present" time="0.503"/>
  <testcase classname="com.myproject.tests.ApiListPageUITest" name="tc_feedback_mail_present" time="0.533"/>
  <testcase classname="com.myproject.tests.ApiListPageUITest" name="tc_logo_present" time="0.481"/>
  <testcase classname="com.myproject.tests.ApiListPageUITest" name="tc_api_list_loads_with_endpoints" time="0.744"/>
  <testcase classname="com.myproject.tests.ApiListPageUITest" name="tc_14_api_endpoints_listed" time="1.034">
    <failure message="Expected at least 14 API endpoints (adjust selector if site differs) expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Expected at least 14 API endpoints (adjust selector if site differs) expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at com.myproject.tests.ApiListPageUITest.tc_14_api_endpoints_listed(ApiListPageUITest.java:98)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- tc_14_api_endpoints_listed -->
  <testcase classname="com.myproject.tests.ApiListPageUITest" name="tc_api_links_open_behavior" time="0.757"/>
  <testcase classname="com.myproject.tests.ApiListPageUITest" name="tc_each_api_entry_details" time="1.276"/>
  <testcase classname="com.myproject.tests.ApiListPageUITest" name="tc_api_versions_displayed" time="1.301"/>
  <testcase classname="com.myproject.tests.ApiListPageUITest" name="tc_footer_copyright" time="2.855"/>
</testsuite> <!-- com.myproject.tests.ApiListPageUITest -->
